.github/workflows/build-umh-core.yml

```
@@ -0,0 +1,187 @@
+
+      - name: Set PLATFORM_SHORT
+        id: platform_short
+        shell: bash
+        run: |
+          PLATFORM_SHORT="${{ matrix.architecture }}"
+          PLATFORM_SHORT="${PLATFORM_SHORT//\//-}"
+          echo "PLATFORM_SHORT=${PLATFORM_SHORT}" >> $GITHUB_OUTPUT
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
```

actionlint (1.7.4)

the runner of "docker/metadata-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)